client/src/main/java/com/linecorp/decaton/client/internal/DecatonClientImpl.java

```diff
@@ -118,6 +127,9 @@ private TaskMetadataProto convertToTaskMetadataProto(TaskMetadata overrideTaskMe
 
         final Long timestamp = overrideTaskMetadata.getTimestamp();
         final Long scheduledTime = overrideTaskMetadata.getScheduledTime();
+        if (overrideTaskMetadata == null) {
```

Comment:
This null check should be above `overrideTaskMetadata.getTimestamp()` .

Otherwise it causes NPE